docker_files/etc/confd/templates/config.json.tmpl

```diff
@@ -0,0 +1,142 @@
+{
+	"wsPort": {{getv "/lisk/wsport" "8001"}},
+	"httpPort": {{getv "/lisk/httpport" "8000"}},
+	"address": "{{getv "/lisk/address" "0.0.0.0"}}",
+	"version": "{{getenv "LISK_VERSION"}}",
+	"minVersion": "{{getenv "LISK_MIN_VERSION"}}",
+	"fileLogLevel": "{{getv "/lisk/fileloglevel" "info"}}",
+	"logFileName": "{{getv "/lisk/logfilename" "logs/lisk.log"}}",
+	"consoleLogLevel": "{{getv "/lisk/consoleloglevel" "error"}}",
+	"trustProxy": {{getv "/lisk/trustproxy" "false"}},
+	"topAccounts": {{getv "/lisk/topaccounts" "false"}},
+	"cacheEnabled": {{getv "/lisk/cacheenabled" "false"}},
+	"wsWorkers": {{getv "/lisk/wsworkers" "1"}},
```

Comment:
Why these variables are all small case? We follow either camel or snake can for readability.

Reply:
`confd` will convert those depeding on the chosen backend: environment variables in our case. That means `/lisk/wsworker` will become `LISK_WSWORKER` and capitalization will get lost.